C_CPP/Competitive Programming/SuperBall_Game_In_C.md

```diff
@@ -0,0 +1,167 @@
+# SUPERBALL GAME
+
+### CODE:
```

Comment:
I will strongly suggest using some code formatter for your code, and also add some relevant comments to make it easy to understand.